src/communication/sns.ts

```diff
@@ -1,5 +1,4 @@
 import { SNS } from 'aws-sdk'
```

Comment:
Can direcly import from SNS module "import SNS from 'aws-sdk/clients/sns"
**NOTE: it's crucial to reduce dependencies module's size(it will reduce lambda's cold start time).

Reply:
actually this is fine for this particular case. As aws-sdk is actually omitted from the stack when deployed on Lambda.

Also kolp listed aws-sdk as peerDep.